src/main/java/Duke.java

```diff
@@ -1,10 +1,325 @@
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
+
+    // shows if the Duke chat bot has been activated
+    private boolean activated;
+    // Line Separator
+    private final String SEP_LINE = "____________________________________________________________";
+    // Standard boot response
+    private final String bootMessage =
+            SEP_LINE
+            .concat(
+            " \n____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n"
+            + "\nMade by Dr-Octavius\n")
+            .concat(SEP_LINE)
+            .concat(
+            "\nHello! I'm Duke\n"
+            + "What can I do for you?\n")
+            .concat(SEP_LINE);
+    // Level-6 -> A-Collections: Text Storage
+    private final List<Task> storage = new ArrayList<>();
+
+    // Duke Constructor
+    public Duke() {
+        this.activated = true;
+    }
+
+    // Checks if Duke chat bot has been activated
+    public boolean isActive() {
+        return this.activated;
+    }
+
+    // Level-1: Greets user when Duke is initialised
+    public void greet() {
+        System.out.println(this.bootMessage);
+    }
+
+    // Level-1: echos user message back to user
+    // Level-5: returns the error message for unknown inputs
+    public String echo() {
```

Comment:
Since the method no longer `echo`s the user input, I believe it would be more fitting to rename it.